webserver/observe/log/src/main/java/module-info.java

```diff
@@ -41,5 +41,5 @@
 
     provides io.helidon.webserver.observe.spi.ObserveProvider
             with io.helidon.webserver.observe.log.LogObserveProvider;
-	
+        
```

Comment:
This change is wrong, please remove it (you may still need to change the copyright, unless you squash your commits locally).